code/parse/sexp.cpp

```diff
@@ -37153,24 +37153,28 @@ SCP_vector<sexp_help_struct> Sexp_help = {
 
 	{ OP_IS_SUBSYSTEM_DESTROYED_DELAY, "Is subsystem destroyed delay (Boolean operator)\r\n"
 		"\tBecomes true <delay> seconds after the specified subsystem of the specified "
-		"ship is destroyed.\r\n\r\n"
+		"ship is destroyed. Subsequent repairs to that subsystem have no effect on this "
+		"operator.\r\n\r\n"
```

Comment:
Perhaps specifying "is destroyed for the first time", would be even clearer. Ditto for the similar ones below.